sympy/assumptions/ask.py

```diff
@@ -292,6 +312,8 @@ def ask(proposition, assumptions=True, context=global_assumptions):
     It should be discerned from :func:`~.refine()` which does not reduce
     the expression to ``None``.
 
+    Unlike :func:~.`refine()`, the return value is not SymPy object.
```

Comment:
This could be made clearer e.g. "this returns True, False or None rather than a symbolic Boolean or some other symbolic object"